journal.py

```diff
@@ -165,19 +174,12 @@ def list_view(request):
 @view_config(route_name='detail', renderer='templates/detail.jinja2')
 def detail_view(request):
     entry = Entry.one(request.matchdict['id'])
-    html_text = markdown(entry.text, output_format='html5')
-
-    def highlighting(matchobj):
-        return highlight(matchobj.group(0), PythonLexer(), HtmlFormatter())
-
-    pattern = r'(?<=<code>)[\s\S]*(?=<\/code>)'
-    html_text = re.sub(pattern, highlighting, html_text)
 
     return {
         "entry": {
             "id": entry.id,
             "title": entry.title,
-            "text": html_text,
+            "text": entry.mkdwn,
```

Comment:
Any specific reason you're decomposing the entry you have into this dictionary?  Why not just hand back `{'entry': entry}`?